src/views/template/warehouse/warehouse.html

```diff
@@ -9,13 +9,13 @@
         </span>
     </div>
     <div>
-        <div class="progress no-border-radius">
+        <div class="progress no-border-radius margin-bottom-0">
             <div class="progress-bar" role="progressbar"
                  aria-valuenow="0" aria-valuemin="0"
                  aria-valuemax="100" style="width:0%"
                  id="warehouse-progress-bar">
             </div>
         </div>
-        <div class="panel-body" id="containers"></div>
+        <div class="panel-body" id="containers" style="max-height: 80vh; overflow-y: scroll;"></div>
```

Comment:
This particular style attribute is used in several places, so it should be a class.